create

```diff
@@ -61,6 +61,17 @@ else
 fi
 
 mke2fs -Fjqt $OSP_FILESYSTEM $filesystem_dev
+
```

Comment:
testing showed that this won't work because it's *after* the `mk2fs` call. it should at least be done *before*.